sdcm/utils/aws_builder.py

```diff
@@ -137,18 +208,144 @@ def associate_elastic_ip(self):
         self.region.client.associate_address(AllocationId=self.elastic_ip.allocation_id,
                                              InstanceId=self.instance.instance_id)
 
-    def configure(self):
+    def configure_one_builder(self):
         self.region.create_sct_ssh_security_group()
         self.create_elastic_ip()
         self.associate_elastic_ip()
         self.add_to_jenkins()
 
+    def create_launch_template(self):
+        LOGGER.info(f"{self.region.region_name}: create_launch_template")
```

Comment:
It makes the precommit fail.
Need to use lazy formatting in the logging lines.

See: https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-5585/58/execution/node/23/log/

Reply:
fixed